src/Sentry/Sentry.csproj

```diff
@@ -12,13 +12,13 @@
   <ItemGroup Condition="'$(TargetFramework)' == 'net461'">
     <PackageReference Include="Newtonsoft.Json" Version="6.0.8" />
     <Reference Include="System.Web" />
+    <Reference Include="System.Net.Http" />
```

Comment:
Do we need to add this reference? Not sure I follow

Reply:
No idea. It started to complain after I removed `System.Collections.Immutable`. Also curious if `<Reference Include="System.Web" />` should be removed.

Reply:
`System.Web` is used because we grab the request data from `HttpContext` when available.
It's odd that `System.Collections.Immutable` would drag `System.net.Http` but since it's just `Reference` it doesn't really matter